<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1506</width>
    <height>466</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(32, 32, 32)</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Chọn kiểu đầu vào </string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textcrawl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>541</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);background-color:rgb(43, 43, 43)</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>1501</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(43, 43, 43);color:rgb(255, 255, 255);
    QHeaderView::section {
        background-color: rgb(43, 43, 43);
        color: white;
    }
</string>
   </property>
   <column>
    <property name="text">
     <string>link page</string>
    </property>
    <property name="background">
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
    </property>
    <property name="foreground">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
   </column>
   <column>
    <property name="text">
     <string>tên page</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>quốc gia</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>follower</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>số lượng video</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>tổng view video</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>số lượng reels</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>tổng views reels</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>video</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string> reels</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="start">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>80</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(43, 43, 43);
color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>quét</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>80</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(43, 43, 43);color:rgb(255, 255, 255)
</string>
   </property>
   <property name="text">
    <string>dừng</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>30</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(43, 43, 43);
color : rgb(255, 255, 255)</string>
   </property>
   <item>
    <property name="text">
     <string>quét full page</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>chỉ quét follow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>quét 10 video/reels gần nhất</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>chỉ quét video</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>chỉ quét reels</string>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>10</y>
     <width>481</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(43, 43, 43)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(43, 43, 43);color:rgb(255, 255, 255)
</string>
   </property>
   <property name="text">
    <string>thông tin</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>40</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(43, 43, 43);color:rgb(255, 255, 255)
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>140</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(43, 43, 43);color:rgb(255, 255, 255)
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>90</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(43, 43, 43);color:rgb(255, 255, 255)
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="resett">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>130</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(43, 43, 43);
color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>lưu cấu hình</string>
   </property>
  </widget>
  <widget class="QComboBox" name="input">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(43, 43, 43);
color:rgb(255, 255, 255);
border-color:rgb(255, 255, 255)</string>
   </property>
   <item>
    <property name="text">
     <string>Link</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UID</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>130</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(43, 43, 43);
color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>load cấu hình</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
